src/core/protocol-authorization.ts

```diff
@@ -413,4 +417,7 @@ export class ProtocolAuthorization {
     }
   }
 
+  private static getRecordDefinitionFromPath(protocolPath: string): string {
```

Comment:
I kept thinking it's the definition that's returned.

```suggestion
  private static getRecordDefinitionIdFromPath(protocolPath: string): string {
```
Could also be more concise if you prefer:
```suggestion
  private static getRecordDefinitionId(protocolPath: string): string {
```

Reply:
fixed